_data/icons/vsl.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url":"ipfs://QmeknNzGCZXQK7egwfwyxQan7Lw8bLnqYsyoEgEbDNCzJX",
```

Comment:
where did you pin this CID? I cannot resolve it

Reply:
I pinned it with pinata
You can check it here https://gateway.pinata.cloud/ipfs/QmeknNzGCZXQK7egwfwyxQan7Lw8bLnqYsyoEgEbDNCzJX